framework/auth/views.py

```diff
@@ -184,9 +184,10 @@ def confirm_email_get(token, auth=None, **kwargs):
                 return redirect(
                     campaigns.campaign_url_for(campaign)
                 )
-            if len(auth.user.emails) == 1:
+            if len(auth.user.emails) == 1 and len(auth.user.email_verifications) == 0:
```

Comment:
This doesn't look correct; won't a new user have exactly one `email_verification`, the one for their primary email?